website/docs/overrides/power-select/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,7 @@
+## Conformance status
+
+<Doc::Badge @type="critical">Non-conformant</Doc::Badge>
+
+This addon does not provide a WCAG-conformant component by default; users should carefully consider their options before use.
+
```

Comment:
Would we want to give them any more details on the issues with it?

Reply:
In the team sync we decided not to.